.editorconfig

```
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{cpp,h,py}]
+charset = utf-8
+indent_style = tab
+indent_size = 4
```

Shouldn't this be `tab_width`?

```suggestion
tab_width = 4
```

---

>tab_width: a whole number defining the number of columns used to represent a tab character. This defaults to the value of indent_size and doesn't usually need to be specified. 

https://editorconfig.org/

Since indent_size is 4 by default and matches tab_width, but we are using tabs here, it makes more sense to specify this as tab_width rather than indent_size.